influxdb/content.md

```diff
@@ -1,643 +1,262 @@
-
-To start an InfluxDB v2 container, enter the following command:
-
-```bash
-docker run \
-    -p 8086:8086 \
-    -v "$PWD/data:/var/lib/influxdb2" \
-    -v "$PWD/config:/etc/influxdb2" \
-    %%IMAGE%%:2
-```
-
-Replace the following with your own values:
+InfluxDB is the time series database platform designed to collect, store, and process large amounts of event and time series data. Ideal for monitoring (sensors, servers, applications, networks), financial analytics, and behavioral tracking.
 
--	`$PWD/data`: A host directory to mount at the container's [InfluxDB data directory](https://docs.influxdata.com/influxdb/v2/reference/internals/file-system-layout/?t=docker#file-system-layout) path
--	`$PWD/config`: A host directory to mount at the container's [InfluxDB configuration directory](https://docs.influxdata.com/influxdb/v2/reference/internals/file-system-layout/?t=docker#file-system-layout) path
+**This README covers all currently supported versions of InfluxDB:**
 
-After the container starts, the InfluxDB UI and API are accessible at http://localhost:8086 on the host. You're ready to set up an initial admin user, token, and bucket from outside or inside the container--choose one of the following:
+-	InfluxDB 3 Core: `influxdb:3-core`
+-	InfluxDB 3 Enterprise: `influxdb:3-enterprise`
+-	InfluxDB v2: `influxdb:2`
+-	InfluxDB v1: `influxdb:1.11`
+-	InfluxDB Enterprise v1: `influxdb:1.11-data` and `influxdb:1.11-meta`
 
--	**Set up InfluxDB from outside the container**: [Set up InfluxDB](https://docs.influxdata.com/influxdb/v2/get-started/setup/) from the host or network using the InfluxDB UI, `influx` CLI, or HTTP API.
+Find a specific InfluxDB version below for startup examples and links to full documentation.
 
--	**Set up InfluxDB from inside the container**: Use `docker exec` to run the `influx` CLI installed in the container--for example:
+## How to use InfluxDB 3 images
 
-	```bash
-	docker exec influxdb2 influx setup \
-	  --username $USERNAME \
-	  --password $PASSWORD \
-	  --org $ORGANIZATION \
-	  --bucket $BUCKET \
-	  --force
-	```
+InfluxDB 3 is the latest version and features Apache Arrow, DataFusion SQL, and object storage for sub-10ms queries and unlimited cardinality. The InfluxDB 3 Core and InfluxDB 3 Enterprise editions both support SQL and InfluxQL queries, include the Python processing engine, and work with the InfluxDB 3 Explorer UI.
 
-See the [`influx setup` documentation](https://docs.influxdata.com/influxdb/v2/reference/cli/influx/setup/) for the full list of options.
+-	**Core** (free, open source): For real-time monitoring and recent data
+-	**Enterprise**: Adds historical data, clustering, HA, and security. Includes 30-day trial.
 
-*If you run setup from within the container, InfluxDB stores `influx` CLI [connection configurations](/influxdb/v2/reference/cli/influx/#provide-required-authentication-credentials) in the container's `/etc/influxdb2/influx-configs` file.*
+### Pull the Image
```

Comment:
This is the kind of extreme level of detail we usually leave out of these image descriptions because we _are_ pretty severely limited on space on Docker Hub (where this description actually ends up), and things like "how to pull a Docker image" are kind of orthogonal / low-level assumed knowledge.

See https://hub.docker.com/_/influxdb for how this image description ends up getting unceremoniously "chopped" thanks to the 25k byte length limit.

(Also, the vast majority of actual users will not run `docker pull` explicitly, but will rather embed this name in a `compose.yaml`, Kubernetes manifest, or `docker run` and it will be pulled incidentally / automatically instead.)

Reply:
7f2c18e